public/doc/openapi-3-v1.0.0.json

```diff
@@ -1110,7 +1110,7 @@
                 "description": "Query by id",
                 "required": true,
                 "schema": {
-                    "type": "string"
+                    "$ref": "#/components/parameters/Ident"
```

Comment:
@hsong-rh can we call this Id or fully spell it out as Identity.
https://github.com/ManageIQ/topological_inventory-api/blob/daa581abc260a37fe08fb3f7bc254cb259e0a6a3/public/doc/openapi-3-v1.0.0.json#L8008